ext/node/polyfills/net.ts

```diff
@@ -393,6 +408,107 @@ function _afterConnect(
   }
 }
 
+function _createConnectionError(req, status) {
```

Comment:
nit: I think we can just remove the underscore prefix for all the functions so they're easy to search in nodejs src too? or is something preventing us from doing that?

Reply:
> I think we can just remove the underscore prefix for all the functions so they're easy to search in nodejs src too?

Sounds reasonable to me, but I would do that in another PR as there's so many of them.

Reply:
https://github.com/denoland/deno/issues/26823